### DISCUSSION TITLE
PPI and Motif data for panda.py

### DISCUSSION THREAD

**Discussion by adinasarapu** [netZoo/netZooR#210] [Q&A] [answered]
Hi, 

Do I need to get my PPI and Motif data based on my samples or Is it OK to use GTEx files provided by you? 

Here is my code.

```
panda1 <- panda.py(expr_file = "Control_Only_TMM_log2.txt" , motif_file = "motif_hg38_refseq.txt", ppi_file = "ppi_GTEx.txt",  modeProcess="legacy", remove_missing = TRUE)
merged.panda <- panda.diff.edges(panda1, panda2, condition_name="control")
vis.diff.panda.in.cytoscape(merged.panda,condition_name = "control", network_name="PANDA_Diff_Edges" )
```

**Comment by marouenbg** [comment, marked as answer]
Hi @adinasarapu ,

Yes, PPI and motif files are not condition-specific therefore you can use generic ones. However, PPI and Motif files can have different coverage, i.e., the number of TFs and genes included in them. For example, TF PPI based on [string V10](https://string-db.org/) would have less TFs included than V11.
You can find PPI and motif files of different coverages in [GRAND](https://grand.networkmedicine.org/) database.

Best,
Marouen

**Reply by adinasarapu** [reply]
Thank you. Very useful.

**Reply by adinasarapu** [reply]
Marouen, 

I have a small network based on differential targeting analysis (vis.diff.panda.in.cytoscape method used).
[PANDA_Diff.pdf](https://github.com/netZoo/netZooR/files/7100083/PANDA_Diff.pdf)
 
**How to interpret this network?** 

Thank you.

**Reply by marouenbg** [reply]
Hi @adinasarapu ,

Really nice network! These are TFs, therefore information about them comes from motif TFs and PPI TFs. Gene expression data only intervenes for target gene inference but not TFs. The reason for this, is that TF activity is not always correlated to their expression because most TFs tend to be active at low expression levels.

However, you can include the TFs that are present in gene expression by filtering motif TFs and PPI TFs to include only the ones you have in gene expression.

All the best,
Marouen

**Reply by adinasarapu** [reply]
Agree. I have filtered PPI and TF-target data with genes present in my expression file. 

Since both control (black edge) and disease (red) edges are targeting the same target gene, how to interpret it (that process). If you find a paper that describes the differential networks interpretation please forward it to me.

Thank you,

**Reply by marouenbg** [reply]
One approach would be to say that the gene has a differential targeting profile in control vs disease. In other words, the modulation of gene expression occurs by new set of TFs in disease. Here is one interesting paper that addresses this question: https://internal-journal.frontiersin.org/articles/10.3389/fgene.2021.649942/full

**Reply by adinasarapu** [reply]
Thank you.
